infra-templates/rbd/0/rancher-compose.yml

```diff
@@ -4,6 +4,7 @@
   description: |
     Docker volume plugin for Ceph RBD
   minimum_rancher_version: v1.6.8-rc1
+  maximum_rancher_version: v1.6.8-rc1
```

Comment:
this needs to be at v1.6.12, otherwise there is no option for the releases

Reply:
@deniseschannon Done.